pkg/ddevapp/config.go

```diff
@@ -933,7 +933,7 @@ func (app *DdevApp) RenderComposeYAML() (string, error) {
 	extraWebContent = extraWebContent + "\nENV NVM_DIR=/home/$username/.nvm"
 	if app.NodeJSVersion != nodeps.NodeJSDefault {
 		extraWebContent = extraWebContent + "\nRUN npm install -g n"
-		extraWebContent = extraWebContent + fmt.Sprintf("\nRUN n install %s", app.NodeJSVersion)
+		extraWebContent = extraWebContent + fmt.Sprintf("\nRUN n install %s && ln -s /usr/local/bin/node /usr/local/bin/nodejs", app.NodeJSVersion)
```

Comment:
Add force flag because someone may link `/usr/local/bin/nodejs` to somewhere else in `pre.Dockerfile`.

```suggestion
		extraWebContent = extraWebContent + fmt.Sprintf("\nRUN n install %s && ln -sf /usr/local/bin/node /usr/local/bin/nodejs", app.NodeJSVersion)
```